tests/conftest.py

```diff
@@ -73,6 +81,16 @@ def samples(app):
         samples_collection.delete_many({})
 
 
+@pytest.fixture
```

Comment:
This is already done in the fixture which creates events.

Reply:
This is for removing events when testing endpoints (which create an event) if we don't use the events fixture because we don't need to have events in database before the request.